scores-src/src/common/formFields/index.tsx

```diff
@@ -49,15 +59,27 @@ export function Field(props: FieldProps) {
   const FieldComponent = props.independent ? FastField : FormikField;
   return (
     <FieldComponent name={props.name}>
-      {({ field, meta }: FormikFieldProps<any>) => (
-        <FieldWrapper {...props}>
-          <Form.Control
-            {...field}
-            type={props.type}
-            isValid={meta.touched && !meta.error}
-            isInvalid={meta.touched && !!meta.error}
-          />
-        </FieldWrapper>
+      {({ field, meta, form }: FormikFieldProps<any>) => (
+        <>
+          {props.type == "number" ? (
+            <NumberInput
+              {...field}
+              value={field.value}
+              /* necessary to properly handle NumberInput onChange having a different param type */
+              onChange={(value) => form.setFieldValue(props.name, value)}
+              error={meta.touched && meta.error}
+              label={props.title}
+            />
+          ) : (
+            <TextInput
```

Comment:
We need to display `props.helper` somewhere - it was handled by the FormWrapper earlier